lib/managers/world_timer.cpp

```diff
@@ -14,6 +14,9 @@ double WorldTimer::get_time_sec() {
     uint64_t time = get_time();
     uint64_t frequency = glfwGetTimerFrequency();
 
+    if (frequency == 0)
+        return 0;
+
```

Comment:
Why?
[The value of zero here indicates the presence of a GLFW error.](https://pub.dev/documentation/glfw3/latest/glfw3_generated_lib_glfw3/glfwGetTimerFrequency.html)
There already are a couple of tripwires for this, but, if you want to add another one, it would be better to setup proper GLFW [error handling system](https://www.glfw.org/docs/3.0/group__error.html).

Reply:
When we call this function before GLFW initialization, GLFW returns zero and we get the correct time (it works, for example, in the initialization of other static variables). If GLFW returns zero while the program is running, this division by zero will cause the whole program to crash. Therefore, it would be better to have a way to check if the current time is zero as an error indicator.

Reply:
It would be better to have a proper error handling system, instead of checking for invalid values everywhere they might occur.

The main problem with current solution is that it is not scalable and is silent, meaning it does not cover other error cases and just silently passes invalid values forward without notifying anyone.

For now this solution will do, but it will have to be fixed in future versions of the project.

Reply:
Oh, wait. Would you rather have a program that fails semi-imidiately in case of an error, or the one that stops the timer and continues working as nothing have ever happened?

I would prefer the first implementation, especially with features like skeletal animation on the way, where the question of "why doesn't the character move" can have many answers, with stopped timer being only one of them.

Reply:
Okay, this is a bad way to indicate errors. I wanted the get_time_sec function to work correctly in the static variable initialization (before the main function), but now I realize that this feature isn't so important and we don't need to add this code here.

Reply:
Anyway, this PR is already useless, since we have a PR that solves the same problem, but has a lot more features to do so.

Reply:
Wow, I can't help but feel that I bullied you into dropping these few lines of code that otherwise useful... Sorry for that.

Anyways, the points I wrote still stand, but this PR is not useless. The second pull request (#25) merges into this branch, meaning it will apply changes to the current implementation, and this very PR is the one that is going to be merged into the main branch.